gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/rest/api/service/InstallationService.java

```diff
@@ -32,6 +32,7 @@ public interface InstallationService {
     String ORPHAN_CATEGORY_UPGRADER_STATUS = "ORPHAN_CATEGORY_UPGRADER_STATUS";
     String API_LOGGING_CONDITION_UPGRADER = "API_LOGGING_CONDITION_UPGRADER";
     String ALERTS_ENVIRONMENT_UPGRADER = "ALERTS_ENVIRONMENT_UPGRADER";
+    String AUDIT_ENVIRONMENT_UPGRADER = "AUDIT_ENVIRONMENT_UPGRADER";
```

Comment:
Should we use plural form for consistency with ALERTS_ENVIRONMENT_UPGRADER ?
Or move ALERTS_ENVIRONMENT_UPGRADER to singular as it has not yet been released ?